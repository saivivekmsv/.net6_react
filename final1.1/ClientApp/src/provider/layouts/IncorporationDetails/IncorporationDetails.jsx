import React from "react";
import { IncorporationDetailsContainer } from "../../containers";

const IncorporationDetails = (props) => {
  return <IncorporationDetailsContainer {...props} />;
};

export default IncorporationDetails;
