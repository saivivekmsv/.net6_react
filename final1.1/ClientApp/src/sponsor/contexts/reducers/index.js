export * from "./app-layout";
export * from "./manage-company";
export * from "./create-plan";
export * from "./manage-eligibility";
export * from "./manage-census";
export * from "./manage-payroll";
export * from "./manage-maintenance";
export * from "./manage-report";
export * from "./manage-mapper";
