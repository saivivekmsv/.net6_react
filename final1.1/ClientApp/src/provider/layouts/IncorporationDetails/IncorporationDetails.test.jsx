import React from "react";
import { BrowserRouter as Router } from "react-router-dom";
import { render } from "@testing-library/react";
import IncorporationDetails from "./IncorporationDetails";
import { ManageCompanyProvider } from "../../contexts";

test("renders learn react link", () => {
  const wrapper = render(
    <ManageCompanyProvider>
      <Router>
        <IncorporationDetails />
      </Router>
    </ManageCompanyProvider>
  );
  expect(wrapper.baseElement).toMatchSnapshot();
});
