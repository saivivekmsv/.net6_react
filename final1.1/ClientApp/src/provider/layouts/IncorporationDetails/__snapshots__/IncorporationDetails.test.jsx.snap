// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<body>
  <div>
    <form
      class="h-100"
    >
      <div
        class="d-flex flex-column manage-company h-100"
      >
        <div
          class="d-flex layout-header align-items-center"
        >
          <div>
            <nav
              aria-label="breadcrumb"
            >
              <ol
                class="breadcrumb"
              >
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/home"
                    >
                      Home
                    </a>
                  </a>
                </li>
                <li
                  class="breadcrumb-item"
                >
                  <a
                    href="#"
                    role="button"
                  >
                    <a
                      href="/companys"
                    >
                      Manage Company
                    </a>
                  </a>
                </li>
              </ol>
            </nav>
            <h5 />
          </div>
          <div
            class="d-flex align-items-center justify-content-center tab-wrapper"
          />
          <div
            class="ml-auto"
          >
            <div>
              <a
                href="/companys"
              >
                <button
                  class="btn btn-secondary"
                  flow="add,save"
                  link="/companys"
                  type="button"
                >
                  Cancel
                </button>
              </a>
                 
              <button
                class="btn btn-primary"
                flow="save,add"
                type="submit"
              >
                Save
              </button>
                 
            </div>
          </div>
          <div
            class="error-banner d-flex align-items-center "
          >
            <div>
              <div
                class="error-heading"
              >
                OOPS!
              </div>
              <div
                class="error-reason"
              >
                Something went wrong! please try again.
              </div>
            </div>
            <div
              class="ml-auto"
            >
              <button
                class="btn btn-link"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-times fa-w-10 "
                  color="#212529"
                  data-icon="times"
                  data-prefix="fal"
                  focusable="false"
                  role="img"
                  style="font-size: 1.5rem;"
                  viewBox="0 0 320 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M193.94 256L296.5 153.44l21.15-21.15c3.12-3.12 3.12-8.19 0-11.31l-22.63-22.63c-3.12-3.12-8.19-3.12-11.31 0L160 222.06 36.29 98.34c-3.12-3.12-8.19-3.12-11.31 0L2.34 120.97c-3.12 3.12-3.12 8.19 0 11.31L126.06 256 2.34 379.71c-3.12 3.12-3.12 8.19 0 11.31l22.63 22.63c3.12 3.12 8.19 3.12 11.31 0L160 289.94 262.56 392.5l21.15 21.15c3.12 3.12 8.19 3.12 11.31 0l22.63-22.63c3.12-3.12 3.12-8.19 0-11.31L193.94 256z"
                    fill="currentColor"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
        <div
          class="layout-content h-100 overflow-auto"
        >
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Address 1
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="address1"
                  type="text"
                  value=""
                />
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Address 2
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="address2"
                  type="text"
                  value=""
                />
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Country
                   
                </label>
                <button
                  class="form-control dropSide disabled dropdown"
                  disabled=""
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                    <option
                      value="USA"
                    >
                      USA
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    USA
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    disabled=""
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
              <div
                class=" sm form-group"
              >
                <label
                  class="form-label"
                >
                  State
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Select
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  City
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="city"
                  type="text"
                  value=""
                />
              </div>
              <div
                class=" xs form-group"
              >
                <label
                  class="form-label"
                >
                  Zip Code
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="postalCode"
                  type="text"
                  value=""
                />
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Phone number
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="phoneNumber"
                  type="phone"
                  value=""
                />
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  E Mail
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="email"
                  type="email"
                  value=""
                />
              </div>
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Company Website
                   
                </label>
                <input
                  autocomplete="nope"
                  class="form-control"
                  name="website"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="line-separator"
          />
          <div
            class="row"
          >
            <div
              class="col"
            >
              <div
                class=" md form-group"
              >
                <label
                  class="form-label"
                >
                  Business type
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                    <option
                      value="1"
                    >
                      LLC
                    </option>
                    <option
                      value="2"
                    >
                      PLC
                    </option>
                    <option
                      value="3"
                    >
                      C-Corp
                    </option>
                    <option
                      value="4"
                    >
                      S-Corp
                    </option>
                    <option
                      value="5"
                    >
                      Partnership
                    </option>
                    <option
                      value="6"
                    >
                      Solo Prop.
                    </option>
                    <option
                      value="7"
                    >
                      Other
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Select
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
              <div
                class=" lg form-group"
              >
                <label
                  class="form-label"
                >
                  Taxed as
                   
                </label>
                <div
                  class="form-control form-button-group "
                >
                  <div
                    class="btn-group"
                    name="taxedAs"
                    role="group"
                  >
                    <button
                      class="btn btn-primary"
                      data-attr="1"
                      type="button"
                    >
                      Corp
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="2"
                      type="button"
                    >
                      S-Corp
                    </button>
                    <button
                      class="btn btn-primary"
                      data-attr="3"
                      type="button"
                    >
                      Partnership or Solo Prop
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="d-flex month-day-form-control"
              >
                <div
                  class=" sm form-group"
                >
                  <label
                    class="form-label"
                  >
                    Fiscal year end
                     
                  </label>
                  <button
                    class="form-control dropSide  dropdown"
                    style="position: relative; display: flex; align-items: center;"
                    type="button"
                  >
                    <select
                      style="visibility: hidden;"
                    >
                      <option
                        value=""
                      >
                        ------select-----
                      </option>
                      <option
                        value="1"
                      >
                        Jan
                      </option>
                      <option
                        value="2"
                      >
                        Feb
                      </option>
                      <option
                        value="3"
                      >
                        Mar
                      </option>
                      <option
                        value="4"
                      >
                        Apr
                      </option>
                      <option
                        value="5"
                      >
                        May
                      </option>
                      <option
                        value="6"
                      >
                        Jun
                      </option>
                      <option
                        value="7"
                      >
                        Jul
                      </option>
                      <option
                        value="8"
                      >
                        Aug
                      </option>
                      <option
                        value="9"
                      >
                        Sep
                      </option>
                      <option
                        value="10"
                      >
                        Oct
                      </option>
                      <option
                        value="11"
                      >
                        Nov
                      </option>
                      <option
                        value="12"
                      >
                        Dec
                      </option>
                    </select>
                    <div
                      class="dropSide-custom-select"
                    >
                      Month
                    </div>
                    <span
                      class="dropSide-custom-select-arrow"
                      style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                    >
                      <span
                        style="transform: rotate(90deg);"
                      >
                        &gt;
                      </span>
                    </span>
                  </button>
                </div>
                   
                <div
                  class=" xs form-group"
                >
                  <label
                    class="form-label"
                  >
                     
                     
                  </label>
                  <button
                    class="form-control dropSide  dropdown"
                    style="position: relative; display: flex; align-items: center;"
                    type="button"
                  >
                    <select
                      style="visibility: hidden;"
                    >
                      <option
                        value=""
                      >
                        ------select-----
                      </option>
                    </select>
                    <div
                      class="dropSide-custom-select"
                    >
                      Date
                    </div>
                    <span
                      class="dropSide-custom-select-arrow"
                      style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                    >
                      <span
                        style="transform: rotate(90deg);"
                      >
                        &gt;
                      </span>
                    </span>
                  </button>
                </div>
              </div>
              <div
                class=" sm form-group"
              >
                <label
                  class="form-label"
                >
                  State of incorporation
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Select
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
              <div
                class=" sm form-group"
              >
                <label
                  class="form-label"
                >
                  Company start date
                   
                </label>
                <button
                  class="form-control dropSide  dropdown"
                  style="position: relative; display: flex; align-items: center;"
                  type="button"
                >
                  <select
                    style="visibility: hidden;"
                  >
                    <option
                      value=""
                    >
                      ------select-----
                    </option>
                  </select>
                  <div
                    class="dropSide-custom-select"
                  >
                    Select
                  </div>
                  <span
                    class="dropSide-custom-select-arrow"
                    style="display: flex; position: absolute; right: 0px; padding: 0px; min-width: 32px; font-size: 24px; outline: none; height: 100%;"
                  >
                    <span
                      style="transform: rotate(90deg);"
                    >
                      &gt;
                    </span>
                  </span>
                </button>
              </div>
              <div
                class=" xs form-group"
              >
                <label
                  class="form-label"
                >
                  Tax EIN
                   
                </label>
                <div
                  class="d-flex"
                >
                  <input
                    autocomplete="off"
                    class="form-control"
                    name="taxEIN"
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
